website_wow/templates/assets.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2017 LasLabs Inc.
+    License GPL-3 or later (http://www.gnu.org/licenses/gpl.html).
+-->
+
+<odoo>
+    <template id="assets_frontend"
+              name="WOW Frontend Assets"
+              inherit_id="website.assets_frontend"
+              >
+        <xpath expr="." position="inside">
+
+            <link href="/website_wow/static/lib/css/animate.css"
+                  rel="stylesheet"
+                  type="text/css" />
+
+            <script src="/website_wow/static/lib/js/wow.js"
+                    type="application/javascript" />
+            <script src="/website_wow/static/src/js/website_wow.view.js"
+                    type="application/javascript" />
+
+            <script src="/website_wow/static/tests/js/website_wow.editor.tour.js"
+                    type="application/javascript" />
+        </xpath>
+    </template>
+    <template id="assets_editor"
+              name="WOW Editor Assets"
+              inherit_id="website.assets_editor"
+              >
+        <xpath expr="." position="inside">
+            <script src="/website_wow/static/src/js/website_wow.editor.js"
```

Comment:
This is required by `website_wow.editor.tour.js`, so not much sense to have both in different assets.

Reply:
It seems that in order to require something you also need to have it included in assets. Removing this yields:

```
warning: Some modules could not be started 
Missing dependencies:    ["website_wow.editor"] 
Non loaded modules:      ["website_wow.tour_editor"] 
Debug:                   {website_wow.tour_editor: {…}}
```

Reply:
I mean that I'd put `website_wow.editor.tour.js` inside `assets_editor` instead of `assets_frontend`, because anyways it will fail in frontend when you're not an editor.

Besides, it could go in demo data to avoid overloading the website.